.buildkite/scripts/test-basic-cluster.sh

```diff
@@ -48,3 +55,6 @@ echo squirrel | "${PATH_TO_RPK_FILE}" topic produce testtopic --brokers "$REDPAN
 
 echo "consuming from topic"
 "${PATH_TO_RPK_FILE}" topic consume testtopic --brokers "$REDPANDA_BROKERS" -v -o :end | grep squirrel || exit 1
+
+echo "testing schema registry"
+for ip_port in $(echo $REDPANDA_REGISTRY | tr ',' ' '); do curl $ip_port/subjects ; done 
```

Comment:
The idea here in the future is to use `rpk registry`, once https://github.com/redpanda-data/redpanda/pull/12669 is released. I'll create a follow-up issue after this PR is merged.